### ISSUE TITLE
parameters for asAssetFile() not present in documentation

### ISSUE THREAD

**Issue by DeepBlue-Dev** [contentful/contentful-management.php#171]
In the documentation for uploads [https://github.com/contentful/contentful-management.php#uploads](url) it uses the asAssetFile without parameters, but this version of the function doesn't exist.

![image](https://user-images.githubusercontent.com/57255515/198071325-ed442b8f-4663-4c76-86bd-8fdbae90b2d7.png)

**Comment by RichardParnabyKing** [comment]
The answer is in the function signature. Give it a filename how you want it to appear in the Media section, and the image's content type / mime type.

For example:
`$asset->setFile('en-US', $upload->asAssetFile('my-awesome-image.png', 'image/png'));`

But yes, docs needs updating.

**Comment by PiX06** [comment]
> The answer is in the function signature. Give it a filename how you want it to appear in the Media section, and the image's content type / mime type.
> 
> For example: `$asset->setFile('en-US', $upload->asAssetFile('my-awesome-image.png', 'image/png'));`
> 
> But yes, docs needs updating.

Thanks very much for that - I figured it out shortly after asking but appreciate the reply!

**Comment by DeepBlue-Dev** [comment]
Hi

First of all sorry for the late reaction. But I am happy to see you have solved it.

> The answer is in the function signature. Give it a filename how you want it to appear in the Media section, and the image's content type / mime type.
> 
> For example: `$asset->setFile('en-US', $upload->asAssetFile('my-awesome-image.png', 'image/png'));`
> 
> But yes, docs needs updating.

Do we have a way to contact the repository maintainers? I haven't created a PR because I saw there were some old PR's still open.

**Comment by PiX06** [comment]
I'm afraid I don't have any way to contact them - it's worth noting though that this is an official Contentful library produced by Contentful themselves so it might be possible to contact them via the https://www.contentful.com/get-in-touch/ options here. Not sure how helpful they would be but maybe worth a try!

**Comment by Sebb767** [comment]
Hey @PiX06 , I've added revisiting this library to our roadmap. I'll fix that soon and also add support for `contentful/core` v4.

**Comment by Sebb767** [comment]
I've fixed the documentation now with [v4.1.0](https://github.com/contentful/contentful-management.php/releases/tag/4.1.0). Basically, you need to supply a name and a mime type like this:

```
$asset->setFile('en-US', $upload->asAssetFile('my-asset-name.png', 'image/png'));
```

We unfortunately can't infer these parameters, as you can simply supply the file contents.
